BoundingBox bbox
int32 class_id